openldap-cluster-ubuntu/config_1_loadSyncProvModule.ldif

```diff
@@ -0,0 +1,4 @@
+dn: cn=module{0},cn=config
```

Comment:
Please create a folder for these files as there are so many of them. It's becoming hard to locate the template json files among these.